lib/global_id/railtie.rb

```diff
@@ -18,12 +18,16 @@ class Railtie < Rails::Railtie # :nodoc:
       default_expires_in = 1.month
       default_app_name = app.railtie_name.remove('_application').dasherize
 
+      app.config.global_id.expires_in = app.config.global_id.fetch(:expires_in, default_expires_in)
+
       GlobalID.app = app.config.global_id.app ||= default_app_name
-      SignedGlobalID.expires_in = app.config.global_id.expires_in ||= default_expires_in
+      SignedGlobalID.expires_in = app.config.global_id.expires_in
```

Comment:
May be nice to stop mutating the app config as well
```suggestion
      SignedGlobalID.expires_in = app.config.global_id.fetch(:expires_in, default_expires_in)
```

Reply:
Yep, much neater!